.travis.yml

```diff
@@ -59,14 +59,15 @@ before_script:
   - ./incubator-openwhisk/tools/travis/setup.sh
 
 script:
-  - cd $TRAVIS_BUILD_DIR/../incubator-openwhisk
-  - ./gradlew install
+#  - cd $TRAVIS_BUILD_DIR/../incubator-openwhisk
+#  - ./gradlew install
   - cd $TRAVIS_BUILD_DIR
   - export BUILD_VERSION="latest"
   - if [ ! -z "$TRAVIS_TAG" ] ; then
         export BUILD_VERSION=$TRAVIS_TAG;
     fi
-  - ./gradlew --console=plain releaseBinaries -PpackageVersion=$BUILD_VERSION
+  - ./gradlew compile -PnativeCompile
```

Comment:
Do all of the releases still get built with this native compile change?

Reply:
no only pick the one for the local env.

Reply:
are there platform specific environments for testing on? since go allows for native modules, this may become necessary at some point.